src/highdicom/seg/enum.py

```diff
@@ -34,6 +34,8 @@ class SpatialLocationsPreservedValues(Enum):
     YES = 'YES'
     NO = 'NO'
     REORIENTED_ONLY = 'REORIENTED_ONLY'
+    """A projection radiograph that has been flipped, and/or rotated by a
```

Comment:
I am wondering to which extent we should document individual fields or rather provide a reference to the relevant section of the standard at the class level. What do you think?

Reply:
These are all terms defined by the standard and I am concerned that we are duplicating a lot of standard text, which is continuously updated.

Reply:
I would argue that documenting the individual fields makes the documentation much more accessible and useful (check out how it appears in the rendered pages compared to before). Further although the standard changes often, these enums are very unlikely to change, and if they do change, they will change in a backwards-compatible way so nothing we put here will be invalidated. If new  fields are added, which may well happen, we will need to update the code anyway...

For me it's a no-brainer for the enums with very simple meanings that are completely obscured by the abbreviations used (e.g. 'F' for 'FOOT' in the `PatientOrientationValues`). With some of the more complicated meanings (e.g. value types and relationship types), I can see an argument for simplifying the docstrings and pointing to the standard. However in these cases I have already provided a link to the standard. Are there particular enums where you think we should remove the docstrings?